sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioUtils.scala

```diff
@@ -538,7 +549,8 @@ object AlluxioUtils extends Logging {
       dataFilters: Seq[Expression]): (FileIndex, Option[Map[String, String]])= {
     val hadoopConf = relation.sparkSession.sparkContext.hadoopConfiguration
     val runtimeConf = relation.sparkSession.conf
-    initAlluxioInfo(conf)
+    initAlluxioInfo(conf, relation.sparkSession.sparkContext.hadoopConfiguration,
```

Comment:
we already have this set to hadoopConf variable above use that. Same thing for runtimeConf

Reply:
done